docs/api/containers.md

```diff
@@ -1 +1,3 @@
 ::: textual.containers
+
+::: textual.widgets.ContentSwitcher
```

Comment:
Strictly speaking `ContentSwitcher` is implemented next to the other widgets and imported from there but it doesn't make much sense to me to have it documented next to the other widgets.
I'm wondering if maybe we should actually have `ContentSwitcher` be imported from `textual.containers`.

Reply:
You may be right, not sure 🤔

Reply:
Filed #3752 for future discussion.